backend/de.metas.manufacturing/src/main/java/org/eevolution/process/PP_Product_BOM_Check.java

```diff
@@ -146,11 +145,8 @@ private void checkProductById(@NonNull final I_M_Product product)
 		updateProductLLCAndMarkAsVerified(product);
 
 		// Get Default BOM from this product
-		final I_PP_Product_BOM bom = productBOMDAO.getDefaultBOM(product).orElse(null);
-		if (bom == null)
-		{
-			throw new AdempiereException("No Default BOM found for " + product.getValue() + "_" + product.getName());
-		}
+		final I_PP_Product_BOM bom = productBOMDAO.getDefaultBOMByProductId(ProductId.ofRepoId(product.getM_Product_ID()))
+				.orElseThrow(() -> new AdempiereException("No Default BOM found for " + product.getValue() + "_" + product.getName()));
```

Comment:
:speaking_head: EN `No Default BOM found for product {0}` => DE `Es wurde keine Standard-Stückliste für das Produkt {0} gefunden`